test_conformance/spirv_new/main.cpp

```diff
@@ -174,9 +176,40 @@ test_status checkAddressWidth(cl_device_id id)
   return TEST_PASS;
 }
 
+void printUsage() {
+    log_info("Reading SPIR-V files from default '%s' path.\n", spvBinariesPath.c_str());
+    log_info("In case you want to set other directory use '%s' argument.\n", spvBinariesPathArg.c_str());
+}
+
 int main(int argc, const char *argv[])
 {
     gReSeed = 1;
+    bool modifiedSpvBinariesPath = false;
+    for (int i = 0; i < argc; ++i) {
+        int argsRemoveNum = 0;
+        if (argv[i] == spvBinariesPathArg) {
+            if (argc == 2) {
```

Comment:
This is not correct. What matters is not the total number of arguments (some of which could be processed by the common parsing) but whether there is another argument available after the current one.